frontend/packages/dev-console/src/components/topology/data-transforms/transform-utils.ts

```diff
@@ -409,6 +409,12 @@ export const getBaseWatchedResources = (namespace: string): WatchK8sResources<an
       namespace,
       optional: true,
     },
+    hpas: {
+      isList: true,
+      kind: 'HorizontalPodAutoscaler',
```

Comment:
nit : can use `HorizontalPodAutoscalerModel.kind`

Reply:
Ah yeah... forgot about this. I had issues originally and was hacking my way around. Forgot I left the string. Thanks!